src/explorer/FolderTreeItem.ts

```diff
@@ -60,13 +60,24 @@ export class FolderTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
     }
 
     public compareChildrenImpl(ti1: AzureTreeItem<ISiteTreeRoot>, ti2: AzureTreeItem<ISiteTreeRoot>): number {
-        if (!(ti1 instanceof LogStreamTreeItem) && ti2 instanceof LogStreamTreeItem) {
-            return 1;
-        }
-        if (ti1 instanceof LogStreamTreeItem && !(ti2 instanceof LogStreamTreeItem)) {
-            return -1;
+        let result: number | undefined = instanceOfCompare(ti1, ti2, LogStreamTreeItem);
+
+        if (result === undefined) {
+            result = instanceOfCompare(ti1, ti2, FolderTreeItem);
         }
-        return ti1.label.localeCompare(ti2.label);
+
+        return result === undefined ? ti1.label.localeCompare(ti2.label) : result;
+    }
+}
+
+// tslint:disable-next-line:no-any
+function instanceOfCompare<T>(ti1: AzureTreeItem, ti2: AzureTreeItem, type1: new (...args: any[]) => T): number | undefined {
+    if (!(ti1 instanceof type1) && ti2 instanceof type1) {

```

Comment:
Why call it type1?  Are there going to be more types to compare?  Variable name seems unclear to me.

Reply:
Oh uh `type` is a reserved keyword so I got a tslint error break or something. Changed it a little